package.json

```diff
@@ -31,6 +31,7 @@
     "accounting": "^0.4.1",
     "classnames": "^2.2.5",
     "dom-scroll-into-view": "^1.2.1",
+    "flystyles": "^2.1.12",
```

Comment:
flystyles shouldn't be a dependency, it should be a devDependencies

Reply:
Ok, let's fix it.